app/assets/javascripts/discourse/app/templates/modal/create-account.hbs

```diff
@@ -147,29 +148,30 @@
               {{/if}}
             </form>
           </div>
-          {{#if showCreateForm}}
-            <div class="modal-footer">
-              {{d-button
-                class="btn-large btn-primary"
-                action=(action "createAccount")
-                disabled=submitDisabled
-                label="create_account.title"
-                isLoading=formSubmitted
-              }}
-
-              {{#unless hasAuthOptions}}
-                {{d-button class="btn-large" id="login-link" action=(route-action "showLogin") disabled=formSubmitted label="log_in"}}
-              {{/unless}}
 
-              <div class="disclaimer">
-                {{html-safe disclaimerHtml}}
-              </div>
+        {{/if}}
+        {{#if showCreateForm}}
```

Comment:
The `{{/if}}` on the line just above is also for `showCreateForm`. If that's correct, those ifs should be merged.